androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java

```diff
@@ -44,20 +51,25 @@
 import static android.support.test.espresso.assertion.ViewAssertions.matches;
 import static org.hamcrest.Matchers.not;
 
-
 @RunWith(AndroidJUnit4.class)
 @LargeTest
 public class AuthenticatorActivityTest {
 
     public static final String EXTRA_ACTION = "ACTION";
     public static final String EXTRA_ACCOUNT = "ACCOUNT";
 
+    private int mResultCode = -2;
+    private int WAIT_LOGIN = 5000;
+
+    private static String errorMessage = "Activity not finished";
```

Comment:
As a general convention, constants are named in uppercase , and receive the both ```static final``` modifiers.